website/docs/language/providers/index.mdx

```diff
@@ -31,16 +31,17 @@ generating random numbers for unique resource names.
 Providers are distributed separately from OpenTofu itself, and each provider
 has its own release cadence and version numbers.
 
-The [Public Terraform Registry](https://registry.terraform.io/browse/providers)
+The [Public OpenTofu Registry](https://github.com/opentofu/registry/tree/main/providers)
 is the main directory of publicly available providers, and hosts
 providers for most major infrastructure platforms.
 
+<!--
```

Comment:
Maybe instead of entirely removing the `Provider Documentation` section, it's a good opportunity to tell our users they need to search the provider's documentation in its repository?

Reply:
:+1: